<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLThreeDViewControllerWidget</class>
 <widget class="QWidget" name="qMRMLThreeDViewControllerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>80</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>3D View Controller</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ctkAxesWidget" name="AxesWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoReset">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="2" column="0">
      <widget class="QToolButton" name="ZoomOutButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Zoom out of the scene by a small amount.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewZoomOut.png</normaloff>:/Icons/ViewZoomOut.png</iconset>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="SpinButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Spin the 3D view.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewSpin.png</normaloff>:/Icons/ViewSpin.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="RockButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Rock the 3D view.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewRock.png</normaloff>:/Icons/ViewRock.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="VisibilityButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Toggle visibility of elements in the 3D view.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/VisibleOn.png</normaloff>:/Icons/VisibleOn.png</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="OrientationMarkerButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Orientation marker</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/OrientationMarker.png</normaloff>:/Icons/OrientationMarker.png</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="ZoomInButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Zoom in on the scene by a small amount.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewZoomIn.png</normaloff>:/Icons/ViewZoomIn.png</iconset>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QToolButton" name="MoreToolButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/SliceMoreOptions.png</normaloff>:/Icons/SliceMoreOptions.png</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="StereoButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Select among stereo viewing options (3DSlicer must be started with stereo enabled to use these features).</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewStereo.png</normaloff>:/Icons/ViewStereo.png</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="RulerButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Show ruler. Only available in orthographic projection mode.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/Ruler.png</normaloff>:/Icons/Ruler.png</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="OrthoButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Toggle between orthographic and perspective rendering in the 3D view.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/ViewPerspective.png</normaloff>
         <normalon>:/Icons/ViewOrtho.png</normalon>:/Icons/ViewPerspective.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="CenterButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Center view</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="ViewLinkButton">
       <property name="toolTip">
        <string>Link 3D views. Synchronizes properties of all 3D views in the same view group.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qMRMLWidgets.qrc">
         <normaloff>:/Icons/LinkOff.png</normaloff>
         <normalon>:/Icons/LinkOn.png</normalon>:/Icons/LinkOff.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionNoStereo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>No Stereo</string>
   </property>
  </action>
  <action name="actionSwitchToRedBlueStereo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Red/blue</string>
   </property>
  </action>
  <action name="actionSwitchToAnaglyphStereo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Anaglyph</string>
   </property>
  </action>
  <action name="actionSwitchToQuadBufferStereo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>QuadBuffer</string>
   </property>
  </action>
  <action name="actionSwitchToInterlacedStereo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Interlaced</string>
   </property>
   <property name="toolTip">
    <string>Switch to Interlaced stereo mode</string>
   </property>
  </action>
  <action name="actionSet3DAxisVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3D cube</string>
   </property>
  </action>
  <action name="actionSet3DAxisLabelVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3D axis label</string>
   </property>
  </action>
  <action name="actionSetLightBlueBackground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light blue background</string>
   </property>
   <property name="toolTip">
    <string>Set light blue background</string>
   </property>
  </action>
  <action name="actionSetBlackBackground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Black background</string>
   </property>
   <property name="toolTip">
    <string>Set black background</string>
   </property>
  </action>
  <action name="actionSetWhiteBackground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>White background</string>
   </property>
   <property name="toolTip">
    <string>Set white background</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="icon">
    <iconset resource="../qMRMLWidgets.qrc">
     <normaloff>:/Icons/ViewCenter.png</normaloff>:/Icons/ViewCenter.png</iconset>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
   <property name="toolTip">
    <string>Center view</string>
   </property>
  </action>
  <action name="actionUseDepthPeeling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use depth peeling</string>
   </property>
   <property name="toolTip">
    <string>Depth peeling is used to render transparent surface models in order</string>
   </property>
  </action>
  <action name="actionSetFPSVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../qMRMLWidgets.qrc">
     <normaloff>:/Icons/ViewFPS.png</normaloff>:/Icons/ViewFPS.png</iconset>
   </property>
   <property name="text">
    <string>Show/Hide frames per second (FPS)</string>
   </property>
  </action>
  <action name="actionSwitchToUserDefinedStereo_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>User Defined 1</string>
   </property>
   <property name="toolTip">
    <string>Switch to user defined stereo mode 1</string>
   </property>
  </action>
  <action name="actionSwitchToUserDefinedStereo_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>User Defined 2</string>
   </property>
   <property name="toolTip">
    <string>Switch to user defined stereo mode 2</string>
   </property>
  </action>
  <action name="actionSwitchToUserDefinedStereo_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>User Defined 3</string>
   </property>
   <property name="toolTip">
    <string>Switch to user defined stereo mode 3</string>
   </property>
  </action>
  <action name="actionOrientationMarkerTypeNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>No orientation marker</string>
   </property>
   <property name="toolTip">
    <string>Hide orientation marker</string>
   </property>
  </action>
  <action name="actionOrientationMarkerTypeCube">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cube</string>
   </property>
   <property name="toolTip">
    <string>Show cube orientation marker</string>
   </property>
  </action>
  <action name="actionOrientationMarkerTypeHuman">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Human</string>
   </property>
   <property name="toolTip">
    <string>Show human-shaped orientation marker</string>
   </property>
  </action>
  <action name="actionOrientationMarkerTypeAxes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Axes</string>
   </property>
   <property name="toolTip">
    <string>Show axes orientation marker</string>
   </property>
  </action>
  <action name="actionOrientationMarkerSizeSmall">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Small</string>
   </property>
   <property name="toolTip">
    <string>Small orientation marker size</string>
   </property>
  </action>
  <action name="actionOrientationMarkerSizeMedium">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
   <property name="toolTip">
    <string>Medium orientation marker size</string>
   </property>
  </action>
  <action name="actionOrientationMarkerSizeLarge">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Large</string>
   </property>
   <property name="toolTip">
    <string>Large orientation marker size</string>
   </property>
  </action>
  <action name="actionRulerTypeNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>No ruler</string>
   </property>
   <property name="toolTip">
    <string>Hide ruler</string>
   </property>
  </action>
  <action name="actionRulerTypeThin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Thin</string>
   </property>
   <property name="toolTip">
    <string>Show thin ruler</string>
   </property>
  </action>
  <action name="actionRulerTypeThick">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Thick</string>
   </property>
   <property name="toolTip">
    <string>Show thick ruler</string>
   </property>
  </action>
  <action name="actionRulerColorWhite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>White ruler</string>
   </property>
  </action>
  <action name="actionRulerColorBlack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Black ruler</string>
   </property>
  </action>
  <action name="actionRulerColorYellow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Yellow ruler</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkAxesWidget</class>
   <extends>QWidget</extends>
   <header>ctkAxesWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qMRMLWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
